### PR TITLE
Update TSC and core contributors

### PR DESCRIPTION
**Pull request by haydentherapper**
Signed-off-by: Hayden Blauzvern <hblauzvern@google.com>

### CODE REVIEW

File: content/trust-security.md [hunk 1]
```diff
@@ -63,7 +51,40 @@ sections:
       imageAsset: ""
       ctaText: ""
       ctaLink: ""
-    subHeader: Technical advisory council
+    column3:
+      header: Priya Wadhwa
+      subHeader: Software Engineer, Chainguard
+      github: https://github.com/priyawadhwa
+      twitter: https://twitter.com/priyawadhwa16
+      text: Priya is a maintainer on sigstore and its infrastructure.
```
Comment (haydentherapper) [inline content/trust-security.md:59]: @priyawadhwa Let me know if you want this updated!

File: content/trust-security.md [hunk 2]
```diff
@@ -63,7 +51,40 @@ sections:
...
+      socialLinks: true
+      imageAsset: ""
+      ctaText: ""
+      ctaLink: ""
+    column4:
+      header: Santiago Torres-Arias
+      subHeader: Professor, Purdue University
+      github: https://github.com/SantiagoTorres
+      twitter: https://twitter.com/TorresAriasS
+      text: Santiago's research focuses on securing the software development life-cycle.
```
Comment (haydentherapper) [inline content/trust-security.md:70]: @SantiagoTorres Let me know if you want this updated!

File: content/trust-security.md [hunk 3]
```diff
@@ -63,7 +51,40 @@ sections:
...
+      socialLinks: true
+      imageAsset: ""
+      ctaText: ""
+      ctaLink: ""
+    column5:
+      header: Trevor Rosen
+      subHeader: Engineering Manager, GitHub
+      github: https://github.com/trevrosen
+      twitter: https://twitter.com/trevrosen
+      text: Trevor is focused on securing the software supply chain.
```
Comment (haydentherapper) [inline content/trust-security.md:81]: @trevrosen Let me know if you want this updated!

File: components/coreContributors.vue [hunk 4]
```diff
@@ -44,7 +44,8 @@ export default {
           disableOnInteraction: true
         },
         names: [
-            'Jake Sanders', 'Asra Ali', 'Priya Wadhwa', 'Carlos Tadeu Panato Junior'
+	    'Ville Aikas', 'Asra Ali', 'Hayden Blauzvern', 'Jason Hall', 'Kenny Leung', 'Dan Lorenc',
```
Comment (haydentherapper) [inline components/coreContributors.vue:47]: List gathered from maintainers across Sigstore repos and alphabetized by last name.

File: content/trust-security.md (hunk 2 above)
Comment (SantiagoTorres) [reply content/trust-security.md:70]: could we do `Assistant Professor` to be on the safe side? thank you!

Comment (haydentherapper) [reply content/trust-security.md:70]: done!

File: content/trust-security.md (hunk 3 above)
Comment (trevrosen) [reply content/trust-security.md:81]: Thanks! A couple edits:

* Let's plz remove my Twitter - it's mostly anger about US politics
* `text:  Trevor leads the Package Security Engineering team at GitHub`

File: static/admin/config.yml [hunk 5]
```diff
@@ -1053,6 +1053,493 @@ collections:
                       default: '',
                       required: false,
                     }
+          - label: 'Five Column Text Grid'
+            name: 'fiveColumnTextGrid'
```
Comment (haydentherapper) [inline static/admin/config.yml:1057]: Anyone an expert here on the website? I added this, and now it's not showing up!

File: content/trust-security.md (hunk 3 above)
Comment (haydentherapper) [reply content/trust-security.md:81]: done and done! and now i'm following you on twitter cause sometimes i enjoy angry US politics

**Review by SantiagoTorres — DISMISSED** [review]

**Review by priyawadhwa — APPROVED** [review]
thanks hayden!

**Comment by haydentherapper** [comment]
@ltagliaferri Do you know how to make the grid in https://deploy-preview-235--sigstore.netlify.app/trust-security look a bit better? I copied over the three column text grid to the two column text grid, and the result was that it still looks a lot like a three grid! I'm looking to remove the space in between the two rows, and minimize the space between the two in the second row.

**Review by trevrosen — APPROVED** [review]

File: components/twoColumnTextGrid.vue [hunk 6]
```diff
@@ -0,0 +1,51 @@
+<template>
+    <section class="py-64 md:py-128 md:px-16" :class="[$route.params.slug === 'trust-security' ? 'bg-purple-dark' : 'bg-white']" data-header-text="text-gray-dark">
```
Comment (ltagliaferri) [inline components/twoColumnTextGrid.vue:2]: Locally, I modified this line to decrease the vertical space between the two column grids, for example:

```
<section class="py-64 md:py-0 md:px-16" ...
```

With 0 in both `.vue` files, it looks like this:


Screen Shot 2022-11-02 at 11 00 46 AM

**Review by ltagliaferri — COMMENTED** [review]
@haydentherapper I checked this out locally and was able to move the two column components closer together by modifying the `.vue` grid files. I put `md:py-` down to `0` in both cases, but I think it's up to aesthetics how far down you would like to go.

This is in both `components/twoColumnTextGrid.vue` and `components/threeColumnTextGrid.vue`:

```
    <section class="py-64 md:py-0 md:px-16" :class="[$route.params.slug === 'trust-security' ? 'bg-purple-dark' : 'bg-white']" data-header-text="text-gray-dark">
```

File: components/twoColumnTextGrid.vue (hunk 6 above)
Comment (haydentherapper) [reply components/twoColumnTextGrid.vue]: Thank you, that looks so much better!

**Comment by haydentherapper** [comment]
@trevrosen Can you re-LGTM and merge?

**Review by ltagliaferri — APPROVED** [review]

**Pull request merged**